ports/render96ex/README.md

```diff
@@ -14,11 +14,12 @@ You must own a legal copy of the Super Mario 64 rom.
 
 ## Game data files needed:
 
-Copy your Super Mario 64 rom as *baserom.us.z64* to the root of the *render96ex* folder. A wizard will then guide you during the first launch of the game to install the assets from the rom and the optional packages. During this installation process if it seems that you can not validate a choice (Yes/No/OK) when a message popups try to press **A**/**B** then **DPAD up/down**.
+Copy your Super Mario 64 rom as *baserom.us.z64* to the root of the *render96ex* folder. The patcher will install the assets from the rom and packages on the first launch.
```

Comment:
The script could check the sha1 of a *.z64 file if found and if there's a match, rename it to baserom.us.z64

Reply:
Good idea. Let me have a look at this :)
(I'm already checking the sha1 on baserom.us.z64 in the install script)